src/vnsw/agent/vrouter/ksync/test/test_vnswif.cc

```diff
@@ -117,9 +117,11 @@ TEST_F(TestVnswIf, intf_inactive) {
 
 // Validate that link-local address is deleted when interface is deleted
 TEST_F(TestVnswIf, intf_delete) {
+    Ip4Address vnet1_address = vnet1_->mdata_ip_addr();
     DeleteVmportEnv(input, 1, false);
+    vnet1_ = 0;
```

Comment:
I'd prefer `NULL` or just delete this line and add a comment before creating `vnet1_address` explaining that vnet1_ is gonna get deleted